spec/services/sign_in/session_revoker_spec.rb

```diff
@@ -0,0 +1,91 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe SignIn::SessionRevoker do
+  let(:session_revoker) do
+    SignIn::SessionRevoker.new(refresh_token: refresh_token,
+                               anti_csrf_token: input_anti_csrf_token,
+                               enable_anti_csrf: enable_anti_csrf)
+  end
+
+  describe '#perform' do
+    subject { session_revoker.perform }
+
+    context 'given a refresh token' do
```

Comment:
Also, can you add specs when `enable_anti_csrf` is passed in here? Again, think it should be a pretty straight copy from `session_refresher_specs`

Reply:
👍